assets/images/Svg.js

```diff
@@ -4,6 +4,7 @@ const { isDevelopment } = require("devtools-config");
 
 const svg = {
   "angle-brackets": require("./angle-brackets.svg"),
+  angular: require("./angular.svg"),
```

Comment:
There is an existing angular.svg in the /assets folder which looked good to me.
Shall I replace it with the one you mentioned?